### DISCUSSION TITLE
network resource requirement

### DISCUSSION THREAD

**Discussion by jjhesk** [Chia-Network/chia-blockchain#2087] [Q&A]
From the context of FAQ, Chia is intend to use for all devices. So the question that I have is rather we need to use static IP and high bandwidth for farming.  I think the answer is no, please let me know if there is a special requirement for static IP and bandwidth resources. thanks!

**Comment by cccat6** [comment]
No requirement for that. Calm down. You just need a stable network but not a high-speed network.
